feature/security/gnsi/authz/feature.textproto

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id {
+  name: "system_authz"
+  version: 1
+}
+
+telemetry_path {
+  path: "/system/grpc-servers/grpc-server/state/authz-policy-version"
```

Comment:
Is the plan for gnsi yang to remain outside of openconfig/public?  If so, we need an issue to update validate_paths check to include this.

Reply:
The plan is to move them into OC main. (a TODO in the PR description, I plan to do that, anything we need to do here? ).

Reply:
OK... this will break the path validation checks since they are invalid paths.  I _think_ this would make every single PR going forward fail this check since it checks all paths.

It might be ideal to comment out all the telemetry_path statements until they are valid paths.

Reply:
I'll keep the current dir for now